README.md

```diff
@@ -78,46 +91,65 @@ source ~/.bashrc
 
 ### hazen-web-app
 
-Clone this repository, create new venv and install:
+Clone this repository, create new virtual environment and install dependencies:
 ```shell
 # clone repo
 git clone https://github.com/GSTT-CSC/hazen-web-app.git
 
 # create new venv
+# you may choose any folder to store files for the virtual environment
 python -m venv hazen-web-app
```

Comment:
when we git clone the web-app it automatically installs it in a folder named hazen-web-app. Might be worth choosing another name for the venv ( like hazen-web-app-venv) to distinguish the two